crates/backend/src/rclone.rs

```diff
@@ -150,12 +148,16 @@ impl RcloneBackend {
         let user = Alphanumeric.sample_string(&mut thread_rng(), 12);
         let password = Alphanumeric.sample_string(&mut thread_rng(), 12);
 
-        let mut rclone_command =
-            split(rclone_command.map_or("rclone serve restic --addr localhost:0", String::as_str))?;
-        rclone_command.push(url.as_ref().to_string());
+        let mut rclone_command = rclone_command.map_or(
+            "rclone serve restic --addr localhost:0".to_string(),
```

Comment:
Would it make sense, to take this hardcoded command out as a constant `RCLONE_COMMAND` to `mod constants {}` in `rclone.rs` so we have this easily accessible and separated from the rest of the code?

Reply:
is changed.